kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/RequestStats.java

```diff
@@ -112,6 +113,12 @@
     )
     private final OpStatsLogger fetchDecodeStats;
 
+    @StatsDoc(
+            name = WAITING_FETCHES_TRIGGERED,
+            help = "number of pending fetches that woke up due to some data produced"
+    )
+    private final Counter waitingFetchesTriggered;
```

Comment:
Could you add this new metric to docs/reference-metrics.md?

Reply:
done